bundle/config/artifact.go

```diff
@@ -40,12 +46,21 @@ func (a *Artifact) Build(ctx context.Context) ([]byte, error) {
 	return cmd.CombinedOutput()
 }
 
-func (a *Artifact) Library() compute.Library {
-	lib := compute.Library{}
-	switch a.Type {
-	case ArtifactPythonWheel:
-		lib.Whl = a.RemotePath
-	}
+func (a *Artifact) NormalisePaths() {
+	for _, f := range a.Files {
+		if f.Libraries == nil {
+			continue
+		}
```

Comment:
Shouldn't this check be on `f.RemotePath != ""` instead? I'm not following why `f.Libraries` is involved here.